src/Xamarin.Android.Build.Tasks/Xamarin.Android.Tooling.targets

```diff
@@ -82,6 +82,8 @@ projects.
     </ResolveSdks>
     <ResolveJdkJvmPath
         JavaSdkPath="$(_JavaSdkDirectory)"
+        MinimumSupportedJavaVersion="$(MinimumSupportedJavaVersion)"
+        LatestSupportedJavaVersion="$(LatestSupportedJavaVersion)"
         Condition=" '$(DesignTimeBuild)' != 'True' And '$(_AndroidIsBindingProject)' != 'True' And '$(AndroidGenerateJniMarshalMethods)' == 'True' And '$(JdkJvmPath)' == '' ">
```

Comment:
How was a project template even hitting this? `$(AndroidGenerateJniMarshalMethods)` would not be `True`, right?

Reply:
:thinking:

Maybe this also needs to be in VS as well?

Reply:
We can't find any place within VS that would need fixing.

However, I *did* find a codepath wherein `<ResolveSdks/>` calls `AndroidSdkInfo`, which eventually calls `JdkInfo.GetKnownSystemJdkInfos()` if/when the JDK path provided to the `AndroidSdkInfo` constructor isn't valid, and *this* is likely responsible for the integration test failure originally reported.